runtime/reference/ts_config_migration.md

```diff
@@ -279,6 +279,9 @@ The compiler hint pattern-matching is quite flexible, it accepts both quoted and
 non-quoted values for the specifier, as well as any whitespace around the equals
 sign.
```

Comment:
I think we should get rid of this paragraph. People should ideally be just providing quotes.

Reply:
👍 ref https://github.com/denoland/deno_graph/pull/444